pyhoma/client.py

```diff
@@ -123,40 +130,49 @@ async def register_event_listener(self) -> str:
 
         return listener_id
 
-    async def fetch_event_listener(self, listener_id: str) -> List[Event]:
+    async def fetch_events(self) -> List[Event]:
         """
         Fetch new events from a registered event listener. Fetched events are removed
         from the listener buffer. Return an empty response if no event is available.
         Per-session rate-limit : 1 calls per 1 SECONDS period for this particular
         operation (polling)
         """
-        response = await self.__post(f"events/{listener_id}/fetch")
+        response = await self.__post(f"events/{self.event_listener_id}/fetch")
```

Comment:
What if `fetch_events` does throw a `NotAuthenticatedException`? How do we register the event listener for future executions from `ha-tahoma`?

Reply:
Yes it what you ask for. It's why in #36 I created a `fetch_devices` method.

Reply:
Looks good, just wanted to understand the flow.